docs/tutorial/quickstart.md

```diff
@@ -1,199 +1,272 @@
 # Quickstart
 
-We're going to create a simple API to allow admin users to view and edit the users and groups in the system.
+We're going to create a simple API to allow admin users to view and edit the
+users and groups in the system.
+
+## Requirements
+
+* [Python][python]
+* [pip][pip] - a tool for installing Python packages
+* [virtualenv][virtualenv] - a tool to create isolated Python environments
 
 ## Project setup
 
-Create a new Django project named `tutorial`, then start a new app called `quickstart`.
+### Create a new Django project named `tutorial`, then start a new app called `quickstart`.
+
+Create the project directory
+
+```text
```

Comment:
I used ``text`` here, because commands such as ``virtualenv`` are not recognized as commands, which leads to somewhat random highlighting.